enforcer-rules/src/test/java/org/apache/maven/plugins/enforcer/TestRequireFilesDontExist.java

```diff
@@ -136,6 +136,34 @@ public void testFileDoesNotExist()
         rule.execute( EnforcerTestUtils.getHelper() );
     }
 
+    @Test
+    public void testFileDoesNotExistSatisfyAny()
+            throws IOException
+    {
+        File f = File.createTempFile( "junit", null, temporaryFolder );
+        f.delete();

```

Comment:
Instead of creating and deleting file we can simply create File object with a wrong path

Reply:
True. I just went with the observed pattern in the test case. To be honest I find this approach quite elegant. This way I really I really know the file does not exist.